cranelift/filetests/filetests/isa/aarch64/arithmetic-run.clif

```diff
@@ -0,0 +1,77 @@
+test run
+target aarch64
+
+; i128 tests
+; TODO: Cleanup these tests when we have native support for i128 immediates in CLIF's parser
+function %i128_const_0() -> i64, i64 {
+block0:
+    v1 = iconst.i128 0
+    v2, v3 = isplit v1
+    return v2, v3
+}
+; run: %i128_const_0() == [0, 0]
+
+; TODO: Blocked by https://github.com/bytecodealliance/wasmtime/issues/2906
```

Comment:
Could we include this test in a separate PR for the iconst issue? Or alternately just update the iconst implementation here (it should only be a few lines of code, I think!).

Reply:
Yeah, I think I prefer including this in the PR for the iconst.

I think changing it here involves changing a bit more than just the emission for Iconst.  [`get_constant`](https://github.com/bytecodealliance/wasmtime/blob/76c6b83f6a21a12a11d4f890490f8acb6329a600/cranelift/codegen/src/machinst/lower.rs#L110) only returns a u64 and [`get_immediate`](https://github.com/bytecodealliance/wasmtime/blob/76c6b83f6a21a12a11d4f890490f8acb6329a600/cranelift/codegen/src/data_value.rs#L15) doesn't support 128bit values yet.

Reply:
Ah, yes, you're right, we need more infrastructure for that; sorry! Happy to have this whole test deferred then.